src/quickgui/qgsquickutils.cpp

```diff
@@ -0,0 +1,71 @@
+/***************************************************************************
+  qgsquickutils.cpp
+  --------------------------------------
+  Date                 : Nov 2017
+  Copyright            : (C) 2017 by Peter Petrik
+  Email                : zilolv at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include <QDesktopWidget>
```

Comment:
What's this?

Reply:
to define type returned from `QApplication::desktop()`